cmd/exporter.go

```diff
@@ -35,11 +35,12 @@ import (
 )
 
 var (
-	address             = flag.String("address", "0.0.0.0:8888", "bind address")
-	metricsPath         = flag.String("metrics-path", "/metrics", "metrics path")
-	enableGPU           = flag.Bool("enable-gpu", false, "whether enable gpu (need to have libnvidia-ml installed)")
-	modelServerEndpoint = flag.String("model-server-endpoint", "", "model server endpoint")
-	enabledEBPFCgroupID = flag.Bool("enable-cgroup-id", true, "whether enable eBPF to collect cgroup id (must have kernel version >= 4.18 and cGroup v2)")
+	address                      = flag.String("address", "0.0.0.0:8888", "bind address")
+	metricsPath                  = flag.String("metrics-path", "/metrics", "metrics path")
+	enableGPU                    = flag.Bool("enable-gpu", false, "whether enable gpu (need to have libnvidia-ml installed)")
+	modelServerEndpoint          = flag.String("model-server-endpoint", "", "model server endpoint")
+	enabledEBPFCgroupID          = flag.Bool("enable-cgroup-id", true, "whether enable eBPF to collect cgroup id (must have kernel version >= 4.18 and cGroup v2)")
+	enableHardwareCounterMetrics = flag.Bool("enable-hardware-counter-metrics", true, "whether expose hardware counter metrics")
```

Comment:
`enableHardwareCounterMetrics` might be misleading
I think it should be `exposeHardwareCounterMetrics`

And, "whether expose hardware counter metrics" -> "whether expose hardware counter as prometheus metrics")